src/com/hms/flexyosisoftconnector/Tag.java

```diff
@@ -159,7 +153,7 @@ public void recordTagValue(String time, String value)
          // store the time in epoch time format as well so we don't need to convert it back later when popping off queue
          point.setEpochTime(epochTimeMilliseconds);
 
-         if(logDuplicateValues || !point.valueEquals(lastDataPoint))
+         if(!point.valueEquals(lastDataPoint))

```

Comment:
watch out here - you probably want to remove the if  here.

Reply:
I'm still concerned that the timestamp would be the same which this would catch. I don't think that would ever be the case but would it hurt to keep this? every 'new' point has a different timestamp

Reply:
DataPoint's .valueEquals() compares the datapoint's value, not the datapoint itself.

Reply:
For checking the timestamp equal you would want to look at .equals()

Reply:
I was looking at the wrong function. I'll change that tomorrow morning